### ISSUE TITLE
test cannot handle additional command line parameters when processing multiple packages

### ISSUE THREAD

**Issue by rnapier** [constabulary/gb#129]
If you are in a specific package directory and run `gb test -ginkgo.noColor`, it passes the extra flag to the test program as expected. But if you move up a level (to a directory with multiple packages under it), and run the same command, you'll get the following for each package:

    flag provided but not defined: -ginkgo.noColor
    Usage of ...

**Comment by davecheney** [comment]
Yeah. This sucks, but I think it's the same for the go tool, not that this
is much of an excuse. This is a side effect that `gb $COMMAND` expands to
`gb $COMMAND ./...`, where `go $COMMAND` expands to `go $COMMAND .`

**Comment by rnapier** [comment]
Sorry; I should have noted that this works fine for the `go` tool. My current build includes `go test ./... -ginkgo.noColor`, which is how I discovered the problem.

**Comment by davecheney** [comment]
Hmm. That really sucks then. I'll take a look.

**Comment by mbueti** [comment]
We're running into a similar issues with our tests. We were previously able to specify ldflags (which we want so that we can select our DB environment based on test/dev/etc). But the most recent versions of gb test balk at this (flag provided but not defined: -ldflags), breaking our test process...
